ui/app/components/diff-version-selector.js

```diff
@@ -57,10 +58,11 @@ export default class DiffVersionSelector extends Component {
     let delta = diffpatcher.diff(rightSideVersionData, leftSideVersionData);
     if (delta === undefined) {
       this.statesMatch = true;
-      this.visualDiff = JSON.stringify(leftSideVersionData, undefined, 2); // params: value, replacer (all properties included), space (white space and indentation, line break, etc.)
+      // params: value, replacer (all properties included), space (white space and indentation, line break, etc.)
+      this.visualDiff = htmlSafe(JSON.stringify(leftSideVersionData, undefined, 2));
     } else {
       this.statesMatch = false;
-      this.visualDiff = jsondiffpatch.formatters.html.format(delta, rightSideVersionData);
+      this.visualDiff = htmlSafe(jsondiffpatch.formatters.html.format(delta, rightSideVersionData));
```

Comment:
The template was using triple curly braces which was triggering [this lint warning](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-triple-curlies.md). Using `htmlSafe` allows to bind the value without triple curly braces.

Reply:
I remember working on this. Nice fix. Triple curly's unless you know what it means generally requires you to look it up, where htmlSafe is at least a little more insightful on what it's doing.